### DISCUSSION TITLE
Is there a way to have paralog_retriever NOT generate a heatmap?

### DISCUSSION THREAD

**Discussion by plitof1** [mossmatters/HybPiper#150] [Q&A] [answered]
I keep getting issues with the image being too big.  Have tried playing with the dpi and length settings, however for our purposes we really don't ever need the heatmap.  Thanks...

**Comment by chrisjackson-pellicle** [comment, marked as answer]
Hi @plitof1,

I've just added a `--no_heatmap` flag for the `hybpiper paralog_retriever` command in version 2.1.8. The bioconda packages should be ready by tomorrow.

Cheers,

Chris

**Comment by plitof1** [comment]
Thank you Chris.

From: Chris Jackson ***@***.***>
Date: Tuesday, June 25, 2024 at 12:12 AM
To: mossmatters/HybPiper ***@***.***>
Cc: plitof1 ***@***.***>, Mention ***@***.***>
Subject: Re: [mossmatters/HybPiper] Is there a way to have paralog_retriever NOT generate a heatmap? (Discussion #150)

Hi @plitof1<https://github.com/plitof1>,

I've just added a --no_heatmap flag for the hybpiper paralog_retriever command in version 2.1.8. The bioconda packages should be ready by tomorrow.

Cheers,

Chris
